smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -249,4 +249,12 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
     {
         return string(abi.encodePacked(_prefix, _suffix));
     }
+
+    /*
+     * @dev: updateProphecyThreshold
+     *       Issues a request for the Oracle to update threshold
+     */
+    function updateProphecyThreshold(uint256 _consensusThreshold) public onlyOperator {
```

Comment:
Could you please add docs that explain why we can't call this function unless the contract is paused?

If someone calls this function while the contract is not paused, this could lead to bad results as the threshold drops, but the user might not receive a payout..

Reply:
Agree, it is better update the threshold during pausing the contract. Will add it in document.